flaml/automl.py

```diff
@@ -827,7 +827,7 @@ def _validate_ts_data(
             dataframe = dataframe.join(y_df)
         if any(ts_col.duplicated()):
             logger.warning(
-                f"Duplicate timestamp values found in timestamp column. "
+                "Duplicate timestamp values found in timestamp column. "
                 f"\n{dataframe.loc[ts_col.duplicated(), dataframe.columns[0]]}"
```

Comment:
don't call `ts_col.duplicated()` twice.
How long does it take to deduplicate? There may be a potential performance issue for large data size.